skiko/src/commonMain/kotlin/org/jetbrains/skiko/SkiaLayerAnalytics.kt

```diff
@@ -42,6 +42,8 @@ interface SkiaLayerAnalytics {
         fun contextInit() = Unit
         fun beforeFirstFrameRender() = Unit
         fun afterFirstFrameRender() = Unit
+        fun beforeFrameRender() = Unit
+        fun afterFrameRender() = Unit
```

Comment:
Common interface is triggered only on AWT

Reply:
There was no intention to provide it for non-jvm targets.

This would require a bigger refactoring.

This PR is only extending what was before.

Reply:
Let's track it somewhere

Reply:
I suggest to don't track until someone need it.